shared/js/background.js

```diff
@@ -230,33 +230,15 @@ chrome.webRequest.onBeforeRequest.addListener(
         // Is this request from the tab's main frame?
         const isMainFrame = requestData.type === 'main_frame' ? true : false
 
-        // Fetch upgrade rule from db (synchronous -- Chrome only)
-        if (utils.isChromeBrowser()) {
-            const url = https.pipeRequestUrl(requestData.url, thisTab, isMainFrame)
-            if (url.toLowerCase() !== requestData.url.toLowerCase()) {
-                console.log('HTTPS: upgrade request url to ' + url)
-                if (isMainFrame) thisTab.upgradedHttps = true
-                thisTab.addHttpsUpgradeRequest(url)
-                return {redirectUrl: url}
-            } else {
-              return
-            }
-
-        // Fetch upgrade rule from db (asynchronous -- Firefox/Mozilla)
+        // Fetch upgrade rule from https module:
+        const url = https.getUpgradedUrl(requestData.url, thisTab, isMainFrame)
+        if (url.toLowerCase() !== requestData.url.toLowerCase()) {
+            console.log('HTTPS: upgrade request url to ' + url)
```

Comment:
Do we want these in production?

Reply:
Agree we probably should remove in production, but doesn't seem critical.

They're useful for debugging and there's a lot of logging in the extension now, so I was following convention. We could do something similar to what we do internally where we strip them out as part of a production build? @laurengarcia do you have an existing plan for how to remove log statements?

Reply:
These log statements in particular go to the background process, so it's not visible in the normal dev panel.

Reply:
Yeah, we can wrap these on a logger fn that strips for production but they are very useful and not really visible to users unless they really jump thru some hoops.